docs/resources/conversation.md

```diff
@@ -11,14 +11,23 @@ Manages a Slack channel
 
 This resource requires the following scopes:
 
+If using `bot` tokens:
 - [channels:read](https://api.slack.com/scopes/channels:read) (public channels)
 - [channels:manage](https://api.slack.com/scopes/channels:manage) (public channels)
+- [channels:join](https://api.slack.com/scopes/channels:join) (adopting existing public channels)
+- [groups:read](https://api.slack.com/scopes/groups:read) (private channels)
+- [groups:write](https://api.slack.com/scopes/groups:write) (private channels)
+
+If using `user` tokens:
+- [channels:read](https://api.slack.com/scopes/channels:read) (public channels)
+- [channels:write](https://api.slack.com/scopes/channels:manage) (public channels)
```

Comment:
user token uses 'write' rather than 'manage', and 'write' includes join permission already.